docs/reference/modules/network.asciidoc

```diff
@@ -35,11 +34,23 @@ in the following table:
 
 |`_local:ipv6_` |Will be resolved to loopback IPv6 addresses
 
-|`_non_loopback_` |Addresses of the first non loopback interface
+|`_site_` |Will be resolved to site-local addresses ("private network")
 
-|`_non_loopback:ipv4_` |IPv4 addresses of the first non loopback interface
+|`_site:ipv4_` |Will be resolved to site-local IPv4 addresses
 
-|`_non_loopback:ipv6_` |IPv6 addresses of the first non loopback interface
+|`_site:ipv6_` |Will be resolved to site-local IPv6 addresses
+
+|`_global_` |Will be resolved to globally-scoped addresses ("publicly reachable")
```

Comment:
Maybe a note "this is probably not what you want. never expose elasticsearch to an unprotected ip."

Today I just feel like erring on the side of over-warning people about security stuff.